File: Cloth-Store-Agency-BE/ct-core-model/src/main/java/com/example/ctcoremodel/ColorsModel.java
package com.example.ctcoremodel;

import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@Data
public class ColorsModel {
    private String id;
    private OptionProductModel optionProduct;
    private Double addition;
    private String image;
    private ProductModel product;
}

File: Cloth-Store-Admin-FE/src/app/core/apis/Dtos/Colors.model.ts
import {OptionModel} from "./Option.model";
import {ProductModel} from "./Product.model";

export class ColorsModel {
  id!: string | null;
  optionProduct!: OptionModel | null;
  addition!: number | null;
  product!: ProductModel | null;
  image!: string | null;
}
